src/components/builder/i18n.tsx

```diff
@@ -122,7 +123,10 @@ export function ComponentTranslations<S extends AnyComponentSchema>({
                 </span>
               </td>
               <td>
-                <div aria-describedby={`component-translation-property-${property}`}>
+                <div
+                  aria-describedby={`component-translation-property-${property}`}
+                  className="offb-table__content--allow-break"
```

Comment:
```suggestion
                  className="offb-table__content offb-table__content--allow-break"
```
proper BEM naming requires the block/element to always be specified, and then you can add zero or more modifiers

I hope I can add the BEM SASS utilities here at some point too since they enforce the proper classnames in the DOM :)